server.go

```diff
@@ -8,15 +8,19 @@ import (
 	"github.com/honeynet/ochi/backend"
 )
 
-//go:embed public
+//go:embed public/*
```

Comment:
Was this done by accident?

Reply:
I was trying to get the folder content. It has the same effect and I later travers the folder structure.